tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38
+envlist = py310
 skipsdist = true
```

Comment:
We could update the flake8 version while here

Reply:
Can you suggest  version? I just made this change to mke the CI pass again.

Reply:
7.0.0 (the latest) works locally here

But you need to move the comments in the ignore section *above* the rule numbers.

```
[testenv:flake8]
deps =
    flake8 == 3.8.3
commands = flake8 .

[flake8]
ignore =
    # line too long
    E501,
    # incompatible with pep8.. "Line break occurred before a binary operator"
    W503,
```